build-tools/packages/build-tools/src/fluidBuild/options.ts

```diff
@@ -317,6 +304,22 @@ export function parseOptions(argv: string[]) {
 			break;
 		}
 
+		// Back compat switches
+		if (arg === "--azure") {
+			options.groups.push("azure");
```

Comment:
nit: Consider adding a warning log saying this is deprecated and to use `-g azure`.

Reply:
Same for the others.

Reply:
done